src/Test.UnitTests.Sarif.Converters/FortifyFprConverterTests.cs

```diff
@@ -88,7 +88,8 @@ public void FortifyFprConverter_GetFailureLevelFromRuleMetadata_ReturnsAppropria
             foreach (KeyValuePair<string, FailureLevel> keyValuePair in expectedInputOutputs)
             {
                 ReportingDescriptor rule = new ReportingDescriptor();
-                rule.SetProperty<string>("Impact", keyValuePair.Key);
```

Comment:
>SetProperty [](start = 21, length = 11)

This is interesting. We were _already_ persisting `Impact` in the rule's property bag. Had I known that, I wouldn't have suggested moving it. I didn't think we had it at all.

Reply:
I did mention it in one of our conversation that Accuracy, Impact and Probability are persisted in Rules.Properties and do I need to move them to DefaultConfiguration.Properties.

Reply:
Ok, I see what's going on. This is test code. We were _not_ persisting `Impact` in product code, as I at first thought. So yes, our decision to put `Impact` in `defaultConfiguration.properties` is good.

What I don't understand then, is why the existing test code was putting `Impact` in the rule's property bag, when the product code was doing no such thing.

---
In reply to: [314902229](https://github.com/microsoft/sarif-sdk/pull/1650#discussion_r314902229) [](ancestors = 314902229)

Reply:
Impact is restored to its original position